tests/bootstrap.php

```diff
@@ -31,10 +33,15 @@
     define('MERCHANT_TOKEN', $config['merchant_token']);
     define('PAYEE_ID', $config['payee_id']);
 
-    $data = json_decode(file_get_contents(__DIR__ . '../composer.json'), true);
+    $data = json_decode(file_get_contents(__DIR__ . '/../composer.json'), true);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function file_get_contents() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5624753)